### ISSUE TITLE
some unicodes freeze the terminal (WSL Bash)

### ISSUE THREAD

**Issue by mknj** [microsoft/terminal#4332]
# Environment
Windows 18362
WSL 18362
Windows Terminal (Preview) Version: 0.8.10091.0
inside WSL: ubuntu 18.04.1 LTS  (https://aka.ms/wsl-ubuntu-1804)

## The following command freezes the terminal

```
cat bad
``` 

## The content of the bad file
🦕🐈🐀🐁🐇🐕🐩🐨 🐿🦔🦇🐍🦅🦉🦆🐓🐔🦃🕊🐣🐤🐥🐦🐧🐋🐳🐬🦈🐟🐠🐡🐙🦑🦐🦀🐚🐌 🐢🦎🐊🙈🙉🙊🎠♘♞🐽🐾👣🎠🏇🐀🐃🐅🐇🐉🐍🐎🐐🐒🐓🐕🐖

## The bad file can be created with the following command
``` 
echo 8J+mlfCfkIjwn5CA8J+QgfCfkIfwn5CV8J+QqfCfkKgg8J+Qv/CfppTwn6aH8J+QjfCfpoXwn6aJ8J+mhvCfkJPwn5CU8J+mg/CflYrwn5Cj8J+QpPCfkKXwn5Cm8J+Qp/CfkIvwn5Cz8J+QrPCfpojwn5Cf8J+QoPCfkKHwn5CZ8J+mkfCfppDwn6aA8J+QmvCfkIwg8J+QovCfpo7wn5CK8J+ZiPCfmYnwn5mK8J+OoOKZmOKZnvCfkL3wn5C+8J+Ro/CfjqDwn4+H8J+QgPCfkIPwn5CF8J+Qh/CfkInwn5CN8J+QjvCfkJDwn5CS8J+Qk/CfkJXwn5CWCg== | base64 -d >bad
```

**Comment by mknj** [comment]
most unicode characters work fine, it seems to be the combination or length of these chararacters that causes the problem

**Comment by vogsphar** [comment]
may be duplicate of  #678

**Comment by j4james** [comment]
This doesn't happen in the master build anymore. I believe it was fixed by PR #4150.

**Comment by zadjii-msft** [comment]
@j4james that sure makes sense to me. Thanks for trying it out! I'll just close this as a dupe then.

/dupe #1360
